src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -46,90 +47,31 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Don't have time to look at this csproj today in detail, but at a glance looks good. Will do a side-by-side comparison tomorrow.